docs/scanner-quickstart.md

```diff
@@ -0,0 +1,150 @@
+
+### Initialize Forta Directory
+
+Initialization creates a private key that will sign the alerts from your scan node.  You must set the $FORTA_PASSPHRASE environment variable or provide the --passphrase flag to the `init` command.
+
+Initialize Forta using the `forta init` command
+
+```
+$ forta init --passphrase <your_passphrase>
+```
+
+This generates a config directory, a private key, and output your address
+
+```
+Scanner address: 0xAAA8C491232cB65a65FBf7F36b71220B3E695AAA
+
+Successfully initialized at /yourname/.forta
+```
+
+Take note of the address. This is the value that will be registered in the scan node registry smart contract.
+
+!!! note "Forta Directory"
+    By default, the forta directory is located in `~/.forta`. If you would like to use a different directory, either set the $FORTA_DIR env var or provide the `--dir` flag to the `init` command. Init command will initialize your Forta configuration and key to this directory.
+
+### Configure config.yml
+
+In your Forta directory, there now is a `config.yml` file. You must configure that file so that your scan node knows how to get its blockchain data.
+
+Set the `scan.jsonRpc` and `trace.jsonRpc` values. If you have your own Ethereum node, you can use that node. Keep in mind that the trace endpoint must support `trace_block`.
+
+!!! note "JSON-RPC APIs"
+    The scan node will request every transaction on a target chain, which can add up to a lot of requests. Ensure your endpoints can accept the appropriate level of traffic.  
+    
+    We suggest running your own ethereum light node for the `scan.jsonRpc` and an Alchemy Growth plan for `trace.jsonRpc` endpoint.  
+
+!!! warning "Public JSON-RPC APIs"
+    While there are public endpoints available for many chains, please note that the quality of an endpoint will drive the quality of a scan node's output.  
+    
+    When Forta node economics are introduced, the quality of a scan node's output will drive rewards and slashing.  We strongly recommend providing your own ethereum light node or using a paid provider when possible.
+
+Example configuration
+
+```yaml
+chainId: 1
+
+# The scan settings are used to retrieve the transactions that are analyzed
+scan:
+  jsonRpc:
+    url: https://mainnet.infura.io/v3/KEY
+
+# The trace endpoint must support trace_block (such as alchemy)
+trace:
+  jsonRpc:
+    url: https://eth-mainnet.alchemyapi.io/v2/KEY
+
+```
+
+## Register Scan Node
+
+Please contact the Forta team and provide your node and once approved we will add your node to the Forta network.
+
+!!! important "Early Decentralization Phase"
+    At this phase, only Forta-approved nodes are allowed onto the network. In later phases, Forta will introduce economic security measures that will allow permissionless scan nodes.
+
+## Run Scan Node
+
+### Start Docker
+
+Ensure Docker is running by running a `docker ps`.  If it is not running, start docker before running Forta.
```

Comment:
suggestion: is running using the command `docker ps` (avoids saying running twice)